src/tile.js

```diff
@@ -342,6 +344,18 @@ $.Tile.prototype = {
         return this.getImage();
     },
 
+    /**
+     * The URL of this tile's image.
+     * @member {String} url
+     * @memberof OpenSeadragon.Tile#
+     * @deprecated
+     * @returns {String}
+     */
+    get url() {
+        $.console.error("[Tile.url] property has been deprecated. Use [Tile.prototype.getUrl] instead.");
```

Comment:
I patterned this after this PR, happy to make any changes here. https://github.com/openseadragon/openseadragon/pull/2148/files#diff-18da25a1c96bbe664a75cf01c5348ca5b54fd9fd938855d0af776d4c5b5a970dR349